### DISCUSSION TITLE
HTML issue in zip file and quarto presentation

### DISCUSSION THREAD

**Discussion by lervin1** [quarto-dev/quarto-cli#6632] [Q&A]
### Description

I am having a problem viewing my html file of my quarto presentation when zipped into a zip file. Instead of looking like a presentation with slides, it is in a weird format. I was wondering if anyone can help me please.
![image](https://github.com/quarto-dev/quarto-cli/assets/143349771/a529f0dd-a91c-45d6-b220-ed49f81b1957)
This is a screenshot of what it looks like when I try to open the html file. It does not look like a regular presentation.

**Comment by mcanouil** [comment]
Are you trying to read it from inside the archive?
Did you use `embed-resources: true`?
